components/input/Input.jsx

```diff
@@ -77,6 +79,7 @@ class Input extends React.Component {
         {icon ? <FontIcon className={style.icon} value={icon} /> : null}
         <span className={style.bar}></span>
         {labelText ? <label className={labelClassName}>{labelText}</label> : null}
+        {hint ? <label className={style.hint}>{hint}</label> : null}
```

Comment:
This is the only thing that sounds a little weird to me. Having a `label` tag element wrapping the hint text does not sound very semantic IMO. Shouldn't it be a span or something similar?